tests/unit/test-cmd/test.run.js

```diff
@@ -221,6 +225,35 @@ describe('run', () => {
     });
   });
 
+  it('can reload when user presses R in shell console', () => {
+    const cmd = prepareRun();
+    const {addonReload} = cmd.options;
+
+    return cmd.run({noReload: false})
+      .then(() => {
+        process.stdin.emit('keypress', 'r', {name: 'r', ctrl: false});
```

Comment:
Given that we are using `mock-stdin` elsewhere, it would be better to use it also to test it here.